### PR TITLE
Fix accessibility and best practices score

### PR DESCRIPTION
**Pull request by ravise5**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--hdfc-bank-prod--aemsites.hlx.live/digital/corporate-credit-cards-application-form?leadId=747674001
- After: https://fixlhs--hdfc-bank-prod--aemsites.hlx.live/digital/corporate-credit-cards-application-form?leadId=747674001

### CODE REVIEW

File: blocks/form/styles.css [hunk 1]
```diff
@@ -7,6 +7,11 @@
   --loader-image-url: url('loader.gif');
 }
 
+a:any-link, b.link, a:any-link:hover, b.link:hover {
+  font-weight: var(--font-weight-bold) !important;
+  font-size: var(--body-sm-font-size) !important;
```
Comment (jalagari) [inline blocks/form/styles.css:12]: Why are you using important, lets change in original source selector

File: scripts/scripts.js [hunk 2]
```diff
@@ -119,6 +119,7 @@ async function loadLazy(doc) {
 function loadDelayed() {
   // eslint-disable-next-line import/no-cycle
   window.setTimeout(() => import('./delayed.js'), 3000);
+  window.setTimeout(() => import('../creditcards/corporate-creditcard/cc.js'), 1200);
```
Comment (jalagari) [inline scripts/scripts.js:122]: Why you have added this in delay? what does CC.js does?

Comment (jalagari) [reply scripts/scripts.js:122]: Please add comment

**Comment by ravise5** [comment]
Raising a new PR. Closing this one

**Pull request closed without merging**